src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/client/prisonSearchApi/PrisonerSearchApiServiceTest.kt

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.prisonSearchApi
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import org.springframework.http.HttpStatusCode
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.ClientResult
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.prisonSearchApi.model.Prisoner
+
+private const val PRISON_NUMBER_1 = "1"
+
+private const val PRISON_NUMBER_2 = "2"
+
+class PrisonerSearchApiServiceTest {
+
+  private val prisonSearchApiClient = mockk<PrisonerSearchApiClient>()
```

Comment:
let's call this variable `prisonerSearchApiClient` as it's easily confused with searching for a `prison` otherwise!